third_party/move/move-compiler-v2/src/env_pipeline/inliner.rs

```diff
@@ -436,20 +463,68 @@ impl<'env> Inliner<'env> {
     }
 }
 
+/// Expression rewriter for lambda spec
+struct LambdaSpecRewriter<'env> {
+    env: &'env GlobalEnv,
+    result_flag: bool,
+}
+
+impl<'env> LambdaSpecRewriter<'env> {
+    fn new(env: &'env GlobalEnv, result_flag: bool) -> Self {
+        LambdaSpecRewriter { env, result_flag }
+    }
+}
+
+impl<'env> ExpRewriterFunctions for LambdaSpecRewriter<'env> {
+    fn rewrite_condition(
+        &mut self,
+        _target: &SpecBlockTarget,
+        cond: &Condition,
+    ) -> Option<Condition> {
+        let mut new_cond = cond.clone();
```

Comment:
When inlining the spec block, `requires` is rewritten into `assert` and `ensures` is rewritten into `assume`, which will be verified in the lifted function.

Reply:
done